content/scout/policy/index.md

```diff
@@ -0,0 +1,172 @@
+---
+title: Get started with policy in Docker Scout
+keywords: scout, supply chain, vulnerabilities, packages, cves, policy
+description: |
+  Policies in Docker Scout lets you define supply chain rules and thresholds
+  for your artifacts, and track how your artifacts perform against those
+  requirements over time
+---
+
+> **Early Access**
+>
+> Docker Scout is in [early access](/release-lifecycle/#early-access-ea). The
+> policy feature is a closed access feature of Docker Scout, and is only
+> available to organizations participating in the early access program for
+> policies.
+>
+> If you're interested in trying out this feature, reach out using the form on
+> the [Docker Scout product page](https://docker.com/products/docker-scout)
+{ .restricted }
+
+In software supply chain management, maintaining the security and reliability
+of artifacts is a top priority. Policy in Docker Scout introduces a layer of
+control, on top of existing analysis capabilities. It lets you define supply
+chain rules for your artifacts, and helps you track how your artifacts perform,
+relative to your rules and thresholds, over time.
+
+Learn how you can use policy to ensure that your artifacts align with your
+desired standards.
+
+## Policy evaluation
+
+When you activate Docker Scout for a repository, image tags that you push are
+[automatically analyzed](../advanced-image-analysis.md). The analysis gives you
+insights about the composition of your images, including what packages that
+they contain and what vulnerabilities that they're exposed to. Policy builds on
```

Comment:
**nitpick:** Not a copy expert or an English major but this reads better for me:
```suggestion
insights about the composition of your images, including what packages
they contain and what vulnerabilities they're exposed to. Policy builds on
```

Reply:
Agreed. Also, I think we should refer to the feature as "Policy Evaluation" instead of just "Policy". Let's change "Policy builds on..." to "Policy Evaluation builds on..."